api/v1beta1/openstackconfigversion_types.go

```diff
@@ -36,7 +36,7 @@ type OpenStackConfigVersionStatus struct {
 //+kubebuilder:subresource:status
 // +kubebuilder:resource:shortName=osconfigversion;osconfigversions
 // +operator-sdk:csv:customresourcedefinitions:displayName="OpenStack Config Version"
-// +kubebuilder:printcolumn:name="Generator",type="string",JSONPath=".spec.ConfigGeneratorName",description="Config Generator Name"
+// +kubebuilder:printcolumn:name="Generator",type="string",JSONPath=".spec.configGeneratorName",description="Config Generator Name"
```

Comment:
Initially I had these things in the same patch. This fixes the name which needs to be camelcase. Not sure how I didn't copy that over to the 2nd review which already landed.